src/tests/ftest/pool/attribute.py

```diff
@@ -53,8 +53,8 @@ def create_data_set():
         data_set = {}
         for index in range(1024):
             size = random.randint(1, 100)
-            key = str(index).encode("utf-8")
-            data_set[key] = get_random_bytes(size)
+            random_string = get_random_bytes(size)
+            data_set[random_string] = random_string
```

Comment:
I believe this masks the problem rather than fixes it.   This will result in pool attribute names and values being set with the same value (`random_string`) and will make it impossible to discern if the pool attribute list is returning the attribute names or values.

Reply:
It is true the attribute name and value are the same however the comparison is made to value, not the key name.

Reply:
I'll change the key names to be the index value, that would be easier to read

Reply:
I see the issue, will fix

Reply:
Fixed, please review.